### DISCUSSION TITLE
query解析问题

### DISCUSSION THREAD

**Discussion by 303182519** [midwayjs/midway#3727] [Q&A]
https://midwayjs.org/docs/controller#query 这里面的文档query 是不是有问题
当 Query String 中的 key 重复时，ctx.query 只取 key 第一次出现时的值，后面再出现的都会被忽略。

比如 GET /user?uid=1&uid=2 通过 ctx.query 拿到的值是 { uid: '1' }。

测试返回的是个数组{ uid: ['1', '2'] }

**Comment by czy88840616** [comment]
看起来是这样，可能是之前的文档延续下来了（之前是 egg）
